programs/server/Server.cpp

```diff
@@ -556,7 +556,7 @@ static void sanityChecks(Server & server)
     try
     {
         if (readNumber("/proc/sys/vm/overcommit_memory") == 2)
-            server.context()->addWarningMessage("Linux memory overcommit is disabled.");
+            server.context()->addWarningMessage("Linux memory overcommit is disabled. Check /proc/sys/vm/overcommit_memory");
```

Comment:
Wouldn't it be nicer to define char* constant in `try block` and use it in both places?

The same for the rest.